java/download-preplanned-map-area/README.md

```diff
@@ -45,11 +47,8 @@ The [Naperville stormwater network map](https://arcgisruntime.maps.arcgis.com/ho
 * `SYNC_WITH_FEATURE_SERVICES` - Changes, including local edits, will be synced directly with the underlying feature services.
 * `DOWNLOAD_SCHEDULED_UPDATES` - Scheduled, read-only updates will be downloaded from the online map area and applied to the local mobile geodatabases.
 
-See [Take a map offline - preplanned](https://developers.arcgis.com/android/latest/guide/take-map-offline-preplanned.htm) to learn about preplanned workflows, including how to define preplanned areas in ArcGIS Online. Alternatively, visit [Take a map offline - on demand](https://developers.arcgis.com/android/latest/guide/take-map-offline-on-demand.htm) or refer to the sample 'Generate Offline Map' to learn about the on-demand workflow and see how the workflows differ.
+See [Take a map offline - preplanned](https://developers.arcgis.com/java/latest/guide/take-map-offline-preplanned.htm) to learn about preplanned workflows, including how to define preplanned areas in ArcGIS Online. Alternatively, visit [Take a map offline - on demand](https://developers.arcgis.com/java/latest/guide/take-map-offline-ondemand.htm) or refer to the sample 'Generate Offline Map' to learn about the on-demand workflow and see how the workflows differ.
```

Comment:
we should leave these weblinks as Android rather than Java: to all intensive purposes they're the same, but Android dev users would prefer to be taken to the Android Guide pages.